addon/components/paper-tabs.js

```diff
@@ -0,0 +1,308 @@
+import Ember from 'ember';
+const { computed, observer, $ } = Ember;
+
+export default Ember.Component.extend({
+  tagName: 'md-tabs',
+
+  /* Settings */
+  dynamicHeight: false,
+  alignTabs: 'top',
+  noInk: false,
+  noInkBar: false,
+  centerTabs: false,
+  stretchTabs: 'auto', // todo: find the best default
+  autoSelect: false,
+
+  classNameBindings: [
+    'primary:md-primary',
+    'dynamicHeight:md-dynamic-height',
+    'shouldStretchTabs:md-stretch-tabs'
+  ],
+  attributeBindings: [
+    'alignTabsAttr:md-align-tabs',
+    'borderBottomAttr:md-border-bottom',
+    'styleAttr:style'
+  ],
+
+  /* Attributes Bindings */
+  alignTabsAttr: computed('alignTabs', function() {
+    return this.get('alignTabs'); // todo safestring
+  }),
+  borderBottomAttr: computed('borderBottom', function() {
+    return this.get('borderBottom') ? 'md-border-bottom' : null;
+  }),
+  styleAttr: computed('heightStyle', function() {
+    return `${this.get('heightStyle')} ${this.get('transitionStyle')}`;
+  }),
+
+  /* Style Bindings */
+  heightStyle: computed('dynamicHeight', 'selectedTab.content.height', 'tabs.[]', 'tabsWrapper.height', function() {
+    if (this.get('dynamicHeight')) {
+      let tabsHeight = this.get('tabsWrapper.height');
+      let selectedTab = this.get('selectedTab');
+      if (selectedTab && (selectedTab.get('content.height') > 0)) {
+        return `height: ${tabsHeight + selectedTab.get('content.height')}px;`;
+      } else {
+        return `height: ${tabsHeight}px;`;
+      }
+    }
+    return '';
+  }),
+  transitionStyle: 'transition: all 0.5s cubic-bezier(0.35, 0, 0.25, 1);',
+
+  /* Logic Bindings */
+  loaded: false, // set after render
+  shouldStretchTabs: computed('stretchTabs', 'canvasWidth', function() {
+    switch (this.get('stretchTabs')) {
+      case 'always':
+        return true;
+      case 'never':
+        return false;
+      default:
+        return (!this.get('shouldPaginate') && (this.get('canvasWidth') <= 600));
+    }
+  }),
+  shouldCenterTabs: computed('centerTabs', 'shouldPaginate', function() {
+    return this.get('centerTabs') && !this.get('shouldPaginate');
+  }),
+  shouldPaginate: computed('loaded', 'noPagination', 'canvasWidth', 'pagingWidth', function() {
+    if (this.get('noPagination') || !this.get('loaded')) {
+      return false;
+    }
+    let canvasWidth = this.get('canvasWidth');
+    let pagingWidth = this.get('pagingWidth');
+    return (canvasWidth - pagingWidth) < 0;
+  }),
+  pagingWidth: computed('tabs.[].id', 'shouldStretchTabs', function() {
```

Comment:
Will fix that too, thanks.

I did not really reviewed all the code, just made sure the demo was working